ci-operator/step-registry/single-node/conf/azure/single-node-conf-azure-commands.sh

```diff
@@ -25,6 +25,11 @@ if not "controlPlane" in cfg:
 
 cfg["controlPlane"]["replicas"] = 1
 
+if not "networking" in cfg:
```

Comment:
I'm not sure if this is desired, it seems the azure builds have been using SDN as the default. Any thoughts on changing this to be default ovn for azure as well @omertuc

Reply:
Discussed with Omer, it was decided to change this to OVN for azure as well